pkg/server/server.go

```diff
@@ -56,6 +58,25 @@ type Config struct {
 
 	// Array of profiles names that will be generated on each profiling tick.
 	ProfilingNames []string
+
+	// Backdate is how long to backdate signed identities to account for clock
+	// skew.
+	Backdate time.Duration
+
+	// SVIDTTL is default time-to-live for SVIDs
+	SVIDTTL time.Duration
+
+	// CATTL is the time-to-live for the server CA. This only applies to
+	// self-signed CA certificates, otherwise it is up to the upstream CA.
+	CATTL time.Duration
+
+	// CASubject is the subject used in the CA certificate
+	CASubject pkix.Name
+
+	// CertsPath is the (optional) path that the server certificates persisted
+	// to. If unset, certificates are ephemeral and regenerated on every
+	// launch.
+	CertsPath string
```

Comment:
Might consider sticking these in a generic `data` directory which can be mandatory, like on the agent... seems that the cases in which you wouldn't specify this are pretty narrow, and storing certs seems to have little downside?

Reply:
That's a great idea. I can do that.